riff-cli/cmd/init.go

```diff
@@ -65,8 +67,8 @@ func Init() (*cobra.Command, *options.InitOptions) {
 	initCmd.PersistentFlags().BoolVar(&initOptions.DryRun, "dry-run", false, "print generated function artifacts content to stdout only")
 	initCmd.PersistentFlags().StringVarP(&initOptions.FilePath, "filepath", "f", "", "path or directory used for the function resources (defaults to the current directory)")
 	initCmd.PersistentFlags().StringVarP(&initOptions.FunctionName, "name", "n", "", "the name of the function (defaults to the name of the current directory)")
-	initCmd.PersistentFlags().StringVar(&initOptions.RiffVersion, "riff-version", utils.DefaultValues.RiffVersion, "the version of riff to use when building containers")
 	initCmd.PersistentFlags().StringVarP(&initOptions.Version, "version", "v", utils.DefaultValues.Version, "the version of the function image")
+	initCmd.Flags().StringVar(&initOptions.InvokerVersion, "invoker-version", "", "the version of invoker to use when building containers")
```

Comment:
This is not really "when building containers" (ie not `docker build`), but rather initializing the function

Reply:
Since it is part of the `Dockerfile` it is only actually used during the build although it is specified here for the 'init' command that creates the `Dockerfile`

Reply:
Ah, maybe I'm misunderstanding this then (now that I'm merging the other PR).

Is this the version of the invoker (ie what ends up in the `FROM` clause) or the version of the invoker custom resource? I now understand it is the former, but this is confusing

Really, what this PR deals with is not an *invoker* CRD, but rather an *initer*. Yes, this is a word.

Reply:
`invoker-version` is the tag of the docker image in the `FROM` clause. Its default value is defined by `.spec.version` in the invoker.yaml.